### DISCUSSION TITLE
第2节课笔记(2班)

### DISCUSSION THREAD

**Discussion by MING-ZCH** [InternLM/Tutorial#102] [笔记]
## 2班第二节课笔记交这里哟~
* 链接形式提交
* 也记得在信息收集文档对应作业处填写链接

**Comment by kishiokon** [comment]
[轻松玩转书生·浦语大模型趣味Demo](https://www.bilibili.com/read/cv29104141/?jump_opus=1)

**Reply by phdbrianlee** [reply]
简书笔记： 趣味demo打卡
https://www.jianshu.com/p/cec27b7086fe?v=1704631081799

**Reply by phdbrianlee** [reply]
Lagent推理正确，但回答错误。

**Comment by kyrie2to11** [comment]
[InterLM L-2: 轻松玩转书生·浦语大模型趣味 Demo](https://zhuanlan.zhihu.com/p/676498904?) 打卡~
ps: 教程好详细

**Comment by pranerd** [comment]
[第二节课笔记](https://blog.csdn.net/gaxzj/article/details/135441527?spm=1001.2014.3001.5502)

**Comment by DarthThomas** [comment]
[第二节笔记](https://share-tianhe.notion.site/1e66d92fedd343bdb7ffe59033183215?pvs=4)

**Comment by RyanWu31** [comment]
https://github.com/RyanWu31/tutorial_notebook/blob/main/README.md

**Comment by qihong2019** [comment]
[第2节 轻松玩转书生·浦语大模型趣味Demo](https://www.processon.com/view/link/659ac467fcbb7a672ed17bab)

**Comment by enkilee** [comment]
[书生·浦语大模型全链路开源体系-Lesson2](https://blog.csdn.net/luhputu520/article/details/135453675)

**Comment by jinxianwei** [comment]
[第二节课程学习](https://blog.csdn.net/jinxianwei1999/article/details/135456396?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135456396%22%2C%22source%22%3A%22jinxianwei1999%22%7D)

**Comment by ZixinxinWang** [comment]
https://blog.csdn.net/Wzxdecsdn/article/details/135433382

**Comment by xulinui** [comment]
https://h90ag9106t.feishu.cn/docx/QJFJd6z8ao57KHxkd7LcVsqpnZb?from=from_copylink

**Comment by songyuc** [comment]
[《轻松玩转书生·浦语大模型趣味Demo》学习笔记](https://blog.csdn.net/songyuc/article/details/135434166)

**Comment by yangff0016** [comment]
[第2节课笔记](https://blog.csdn.net/weixin_42837310/article/details/135466066?spm=1001.2014.3001.5502)

**Comment by zyy0831** [comment]
第二课笔记
https://www.yuque.com/yuanyang-xb8hd/eeag2y/nv8avtmn87kn2bus

**Comment by yanxiaoqi932** [comment]
https://wspqhnu603.feishu.cn/docx/E6HMdFuEmotWiQxeNpmcFHkznRd?from=from_copylink

**Comment by Mr-Vincenyang** [comment]
[第二节课笔记](https://zhuanlan.zhihu.com/p/676509882)抱歉，弄晚啦

**Comment by WanpengXu** [comment]
因期末考试 ddl 耽误了，请 TA 评阅，谢谢～
[书生·浦语大模型实战营-第二节](https://b07ofnm9xj7.feishu.cn/wiki/SaBQwkCy7iZRTakm9AncNU2xnyb)

**Comment by KylinCarol** [comment]
https://blog.csdn.net/m0_65412000/article/details/135442260

**Comment by CoderJackZhu** [comment]
[第二节笔记和作业](https://zhuanlan.zhihu.com/p/676665482?)

**Comment by YanxingLiu** [comment]
第二节课笔记 https://s5eixc6gba.feishu.cn/docx/KqWAdfxlDonQ1fxGyAQcGz0Qnrc?from=from_copylink

**Comment by PrinceRay7** [comment]
第二节课笔记
https://blog.csdn.net/Prince_Ray/article/details/135830790?spm=1001.2014.3001.5501
